src/AddTodoForm.js

```diff
@@ -9,8 +9,8 @@ function AddTodoForm({ onAddTodo }) {
     setTodoTitle(newTodoTitle);
   }
 
-  function handleAddTodo(event) {
-    event.preventDefault();
+  function handleAddTodo() {
+    // event.preventDefault();
```

Comment:
Sorry about this!! I made a mistake, you. actually need this :( That's my fault.
You can remove this comment and your app should work as expected 👍